operator/src/network/controller.rs

```diff
@@ -795,20 +795,31 @@ async fn update_peer_status(
         }
     };
 
-    // Determine the status of each peer
+    let keramik_peers: BTreeSet<&PeerId> = status.peers.iter().map(|peer| peer.id()).collect();
+
+    // Query each Keramik peer's connected peers
     let mut min_connected_peers = None;
     for peer in &status.peers {
-        let peer_status = match cx.rpc_client.peer_status(peer.ipfs_rpc_addr()).await {
+        let connected_peers = match cx.rpc_client.connected_peers(peer.ipfs_rpc_addr()).await {
             Ok(res) => res,
             Err(err) => {
                 warn!(%err, peer = peer.id(), "failed to get peer status for peer");
                 continue;
             }
         };
-        debug!(peer = peer.id(), ?peer_status, "peer status");
+        // Filter out connected Keramik peers since we only care about Keramik peers for the purpose of bootstrapping
```

Comment:
I agree, the wording does sound confusing. What do you think of this? I can also use "collect connected Keramik peers".
```suggestion
        // Filter out non-Keramik peers since we only care about connected Keramik peers for the purpose of bootstrapping
```

Reply:
That's great, a lot more clear! And TIL about the "suggestion" feature and how you can commit it 🤯

Reply:
Haha, yes! There's an even cooler feature that I also only learned recently, which is that you can shift-select multiple lines for a multi-line suggestion. Very useful when suggesting a change to a block of code.